wgsl/index.bs

```diff
@@ -9627,6 +9662,38 @@ Each is described in detail in subsequent sections.
       [=group_count_y=] - 1, [=group_count_z=] - 1).
 </table>
 
+##### `subgroup_invocation_id` ##### {#subgroup-invocation-id-builtin-value}
+
+<table class='data'>
+  <tr><td style="width:10%">Name
+      <td><dfn noexport dfn-for="built-in values">subgroup_invocation_id</dfn>
+  <tr><td style="width:10%">Stage
+      <td>[=compute shader stage|compute=] or [=fragment shader stage|fragment=]
+  <tr><td style="width:10%">Type
+      <td>u32
+  <tr><td style="width:10%">Direction
+      <td>Input
+  <tr><td style="width:10%">Description
+      <td>
+      The current invocation's [=subgroup invocation ID=].
+</table>
+
+##### `subgroup_size` ##### {#subgroup-size-builtin-value}
+
+<table class='data'>
+  <tr><td style="width:10%">Name
+      <td><dfn noexport dfn-for="built-in values">subgroup_size</dfn>
+  <tr><td style="width:10%">Stage
+      <td>[=compute shader stage|compute=] or [=fragment shader stage|fragment=]
+  <tr><td style="width:10%">Type
+      <td>u32
+  <tr><td style="width:10%">Direction
+      <td>Input
+  <tr><td style="width:10%">Description
+      <td>
+      The current invocation's [=subgroup size=].
```

Comment:
Is that a per-invocation value? Can we at least guarantee that it is constant inside of a single workgroup, or even better, any execution of the same pipeline?

Reply:
It is detailed below, maybe just removing the word invocation would help, or "The current invocation's subgroup's size" (notice the added "'s")